Jenkinsfile

```
@@ -729,6 +730,23 @@ pipeline {
                         buildHipClangJobAndReboot( build_type: 'debug', setup_flags: NOMLIR_flags, build_cmd: NOMLIR_build_cmd, test_flags: ' --verbose ')
                     }
                 }
+                stage('Fp32 Hip Debug NOCK AnyGPU Build-Only') {
+                    when {
+                        beforeAgent true
+                        expression { params.TARGET_VEGA20 || params.TARGET_VEGA10 || params.TARGET_GFX908 || params.TARGET_GFX90A }
```

[R] Add `TARGET_ANYGPU` and use it here **_and everywhere_**.
```suggestion
                        expression { params.TARGET_ANYGPU }
```
Use case: The user wants to avoid usage of any gfx908 nodes. Clearing the `params.TARGET_GFX908` flag does not _guarantee_ this because ANYGPU stages occupy the 1st available node, which could be gfx908. Disabling ANYGPU stages is only possible via disabling Vega, gfx908 and gfx90a targets, which is not what the user wants.

With the new flag, the user can clear `params.TARGET_GFX908` and `params.TARGET_ANYGPU` and obtain the desired effect.

---

Hold off on this one. While we are adding targets, I'll make sure they do not immediately becomes CI bottleneck.